server/service/src/sync/program_schemas/patient_ui_schema.json

```diff
@@ -378,7 +411,7 @@
                       ["DIVORCED", "Divorced"],
                       ["WIDOWED", "Widowed"],
                       ["SEPARATED", "Separated"],
-                      ["REGISTERED_PARTNERSHIP", "Registered Partnership"]
+                      ["REGISTERED_PARTNERSHIP", "Cohabiting"]
```

Comment:
You're just changing the UI? You don't think we need to update the enum values in the schema too? (Probably not, but just checking)

Reply:
The CAPITAL entries are the predefined values and the UI just displays what we need for this specific case. So we don't need to update the schema...